installer.sh

```diff
@@ -377,59 +381,59 @@ if [[ "$TPM_VERSION" -eq "1" ]] ; then
-    # by directly communicating with the TPM (though not recommended) by setting:
-    # for swtpm2 emulator:
-    #   export TPM2TOOLS_TCTI="mssim:port=2321"
-    # for chardev communication:
-    #   export TPM2TOOLS_TCTI="device:/dev/tpm0"
-    #
-    # sudo useradd --system --user-group tss
-    # git clone https://github.com/tpm2-software/tpm2-abrmd.git tpm2-abrmd
-    # pushd tpm2-abrmd
-    # ./bootstrap
-    # ./configure --with-dbuspolicydir=/etc/dbus-1/system.d \
-    #             --with-systemdsystemunitdir=/lib/systemd/system \
-    #             --with-systemdpresetdir=/lib/systemd/system-preset \
-    #             --datarootdir=/usr/share
-    # make
-    # sudo make install
-    # sudo ldconfig
-    # sudo pkill -HUP dbus-daemon
-    # sudo systemctl daemon-reload
-    # sudo service tpm2-abrmd start
-    # export TPM2TOOLS_TCTI="tabrmd:bus_name=com.intel.tss2.Tabrmd"
-    #
-    # NOTE: if using swtpm2 emulator, you need to run the tpm2-abrmd service as:
-    # sudo -u tss /usr/local/sbin/tpm2-abrmd --tcti=mssim &
 
-    echo
-    echo "=================================================================================="
-    echo $'\t\t\t\tBuild and install tpm2-tools'
-    echo "=================================================================================="
-    git clone $TPM2TOOLS_GIT tpm2-tools
-    pushd tpm2-tools
-    git checkout $TPM2TOOLS_VER
-    ./bootstrap
-    ./configure --prefix=/usr/local
-    make
-    make install
-    popd # tpm
+elif [[ "$TPM_VERSION" -eq "2" ]] ; then
+    if [[ "$BUILD_TPM2_SOFTWARE" -eq "1" ]] ; then
+        echo "=================================================================================="
+        echo $'\t\t\t\Building tpm2-tss'
+        echo "=================================================================================="
+        # TPM2_TSS
+        git clone $TPM2TSS_GIT tpm2-tss
+        pushd tpm2-tss
+        git checkout $TPM2TSS_VER
+        ./bootstrap
+        ./configure --prefix=/usr
+        make
+        make install
+        popd # tpm
+        echo
+        echo "=================================================================================="
+        echo $'\t\t\t\tBuild and install tpm2-tools'
+        echo "=================================================================================="
+        git clone $TPM2TOOLS_GIT tpm2-tools
+        pushd tpm2-tools
+        git checkout $TPM2TOOLS_VER
+        ./bootstrap
+        ./configure --prefix=/usr/local
+        make
+        make install
+        popd # tpm
+        echo "=================================================================================="
+        echo $'\t\t\t\tBuild and install tpm2-abrmd'
+        echo "=================================================================================="
+        useradd --system --user-group tss
+        git clone https://github.com/tpm2-software/tpm2-abrmd.git tpm2-abrmd
+        pushd tpm2-abrmd
```

Comment:
We need to git checkout the correct version of abrmd that is compatible with the TSS used above. Note that this block of code was commented out and was intended to be used as a guide. It might need changes to get it working, and definitely won't work on Fedora (since we are not installing the SELinux policy).

Reply:
agree.

I am throwing these changes up, but yet to test them. I will be running full tests before letting these even get close to being merge worthy (against both Ubuntu and Fedora)

I am just discussing with Javier what the issue might be with SELinux (looks like it might be upstream).